tests/transformers/models/test_causal_lm_models.py

```diff
@@ -32,8 +32,7 @@
     "hakurei/gpt-j-random-tinier",
     "mistralai/Mixtral-8x7B-Instruct-v0.1",
     "TheBloke/TinyLlama-1.1B-Chat-v0.3-AWQ",  # AWQ model
-    #    "TheBloke/Llama-2-7B-Chat-GPTQ",  # GPTQ model -> Enable once GPTQ+ROPE
-    #    issue is resolved
+    "TheBloke/Llama-2-7B-GPTQ",  # GPTQ model
```

Comment:
It doesn't make sense for this to be here, it's out of context of this PR.
Please make separate PR for such cases in future

Reply:
Sure